packages/core/src/module.js

```diff
@@ -160,6 +160,9 @@ export default class ModuleContainer {
 
         message = message.replace(/{name}/g, src)
 
+        // add this extra warning that points to the original error for debuging purposes
+        consola.warn('originalError:', error)
+
         if (this.options._cli) {
           throw new Error(message)
```

Comment:
Nice improvement! Maybe here instead we can throw original error (stack + message) but only refine error.message to add additional hints?

Reply:
I didn't dive in too deep to see what conventions do you use or how are you guys handling errors in general, so I guess, there is a lot of room for improvement :). 
Also, I think that there are other cases as well where the original error is shadowed and would be nice to be exposed.

If you feel that this small improvement needs further discussion/refactoring, maybe let's create an issue and move this fix there.  

Meanwhile, it's up to you if this fix can be merged.

Cheers ✌️

Reply:
Indeed. Thanks for bringing it to top. I will push changes to this PR if is okay ☺️

Reply:
Sounds awesome! Thanks!